wiki/Ranking_Criteria/en.md

```diff
@@ -146,7 +146,8 @@ All forms of Artist/Title standardisation apply to both the `Romanised` and `Uni
 
 - **Logos should not be used as references for capitalisation of titles or artists.** Because logos are often stylised, apply standard capitalisation unless other textual metadata supports it.
 - **Tracks created by artists belonging to doujin circles should list the circle's name as the main artist.** The exception to this is when the artist(s) of a given track is well-known enough by their own name. In this case, the specific artist name(s) may be used instead.
-- **If the same song exists in the Ranked section already, the metadata should be followed unless it breaks other rules in the Ranking Criteria or the Official Sources state something completely different.**
+- **If the same song exists in the Ranked or Loved sections already, the metadata should be followed unless it breaks other rules in the Ranking Criteria or the Official Sources state something completely different.**
```

Comment:
i dont understand why Official Sources is in title case but it was like this already so /shrug

Reply:
maybe because it's a glossary term, but not even the glossary capitalizes it like that and we don't do that for other glossary terms, oop